src/3box.js

```diff
@@ -125,6 +129,11 @@ class Box {
           syncPromises.push(this.public._sync(data.numEntries))
           hasResponse[pubStoreAddress] = true
         }
+        if (data.odbAddress.includes('space') === true) {
```

Comment:
😂 these are always fun.

Also, remove the console log on the next line.

Reply:
👍

Reply:
Oh, haha! I didn't think of that either :P